numpy/random/_examples/cython/setup.py

```diff
@@ -12,7 +12,11 @@
 from os.path import join, dirname
 
 path = dirname(__file__)
+src_dir = join(dirname(path), '..', 'src')
 defs = [('NPY_NO_DEPRECATED_API', 0)]
+inc_path = np.get_include()
+# not so nice. We need the random/lib library from numpy
+lib_path = join(np.get_include(), '..', '..', 'random', 'lib')
```

Comment:
Should we invent a better way to spell this? I am not sure, and if it is `np.get_include(component=None)` with the only other component being `"random"` or similar.

Reply:
This is for the `lib` path, not the include path. Should I add `np.get_libpath('npymath'/'npyrandom')`?